django/middleware/security.py

```diff
@@ -52,4 +53,13 @@ def process_response(self, request, response):
                 if isinstance(self.referrer_policy, str) else self.referrer_policy
             ))
 
+        if self.feature_policy:
+            response.setdefault('Feature-Policy', '; '.join(
+                '%s %s' % (directive, ' '.join(
+                    ('%r' if i in {'none', 'self'} else '%s') % i for i in (
+                        allowlist.split(' ') if isinstance(allowlist, str) else allowlist
+                    )
+                )) for directive, allowlist in sorted(self.feature_policy.items())
```

Comment:
If the Django version which includes this requires Python>=3.6, the dict iteration order is deterministic (insertion order). In this case, is the sort needed?

Reply:
It doesn't harm, but yes, it isn't necessary so I'll remove it.